### DISCUSSION TITLE
Modal vs Full page Creation pattern (Thurs, Dec 17)

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#1043] [Office hours] [answered]
Identity team will be working on integrating the regional efforts on the API Keys and Credentials pages. I'd appreciate if the team can provide feedback on the latest visual explorations I have created using Paste. One bigger decision that is pending is whether to use a combination of modal/full page vs full page for the creation pattern. 

Designs can be found here in [Abstract](https://share.goabstract.com/de36d79a-e50a-4c89-b01f-78208fb1e99d). Will do a walkthrough during office hours.

**Comment by corinne-nagel** [comment, marked as answer]
Hi @pguptaDS , thanks for sharing, the team will review and we can discuss your latest visual explorations Thursday. Is there anything specifically you'd like us to focus on?

The creation pattern/modal guidance work is being prioritized at the moment. If you would like to take on that pattern, please consider contributing a pattern and reach out to DSYS! https://paste.twilio.design/patterns/contribute

**Reply by serifluous** [reply]
Nudge: @pguptaDS, Can you share some specific questions, so we know what to focus on as we review async?

**Reply by pguptaDS** [reply]
I left a comment on this [artboard](https://share.goabstract.com/de36d79a-e50a-4c89-b01f-78208fb1e99d?collectionLayerId=80000084-36cf-4dab-91f4-1010dd501876&mode=design) with my question.

Also, as much as I'd love to, I don't think I have the bandwidth to commit to taking on a pattern right now. However, I do need guidance on a general direction for the creation pattern that I can use in these flows and the team will need a decision now since we have to ship this in Q1 for the regional beta. I'd love to focus our discussion on that during office hours.

Apart from that, any general comments on the usage of various patterns or layout are welcome!
